apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -287,6 +287,28 @@ export default class Utilities {
     });
   }
 
+  public static executeShellCommand(
```

Comment:
>executeShellCommand [](start = 16, length = 19)

We already have executeCommand(), executeCommandAndCaptureOutput(), and executeCommandAsync().  Why do we need ANOTHER function for executing commands?

FYI the NodeJS spawn() sucks, and I have spent a lot of time fixing bugs related to these functions.  The handling of escapes is still not 100% correct.  My plan is to write a bunch of unit tests that capture all the difficult scenarios, and then migrate this code into a **node-core-library** API something like this:

class Process {
  public execute(command, args, options): void;
  public executeAsync(command, args, options): void;
}